packages/kbn-unified-data-table/src/components/data_table.test.tsx

```diff
@@ -1125,17 +1127,19 @@ describe('UnifiedDataTable', () => {
     it(
       'should not allow comparison if less than 2 documents are selected',
       async () => {
-        await renderDataTable({ enableComparisonMode: true });
+        renderDataTable({ enableComparisonMode: true });
```

Comment:
```suggestion
        await renderDataTable({ enableComparisonMode: true });
```
This call should be awaited

Reply:
Thanks for spotting this! Updated in [bf0438c](https://github.com/elastic/kibana/pull/189949/commits/bf0438cd67c88645fe112f6011ce9f182d1812bb).